config/environments/development.rb

```diff
@@ -43,8 +43,12 @@
   config.auth0_client_secret = ENV['MUMUKI_AUTH0_CLIENT_SECRET']
   config.auth0_domain = ENV['MUMUKI_AUTH0_DOMAIN']
 
-  config.offline_mode = ENV['OFFLINE_MODE']
-  config.domain_url = ENV['DOMAIN_URL'] || 'localmumuki.io:3000'
-  config.queueless_mode = ENV['QUEUELESS_MODE']
-  config.cookies_domain = ENV['COOKIES_DOMAIN'] || '.localmumuki.io'
+  config.offline_mode = ENV['MUMUKI_OFFLINE_MODE']
```

Comment:
Don't forget to update [this](https://github.com/mumuki/mumuki-offline/blob/236a3c273be2d36bf1b0869aebeffc5be88090ae/scripts/env.sh#L14) at merge